css-view-transitions-2/Overview.bs

```diff
@@ -598,20 +615,19 @@ When capturing the old or new state for an element, perform the following steps
 			Note: this means that calling {{Document/startViewTransition()}} while capturing the old document for a cross-document view-transition would run the callback but skip the animation.
 
 		1. Let |outboundTransition| be a new {{ViewTransition}} object in |oldDocument|'s [=relevant Realm=],
-			whose [=ViewTransition/active types=] is |resolvedRule|, and whose [=ViewTransition/process old state captured=] is set to the following steps:
+			whose [=ViewTransition/active types=] is |resolvedRule|.
 
-			Issue: should we check for the opt-in again, in case there was a CSSOM change in a requestAnimationFrame callback?
+		1. Run the following steps when [=Perform pending transition operations|performing pending transition operations=]:
 
-			1. If |outboundTransition|'s [=ViewTransition/phase=] is "`done`", then call |onReady| and return.
+			1. If |outboundTransition|'s [=ViewTransition/phase=] is "`done`", then call |proceedWithNavigation| and return.
 
 			1. Assert: |outboundTransition|'s  [=ViewTransition/phase=] is "`pending-capture`".
 
-			1. [=Clear view transition=] |outboundTransition|.
+			1. Set |oldDocument|'s [=auto-skip view transitions=] to false.
 
-				Note: The ViewTransition object on the old Document should be destroyed after its state has been copied to the new Document below.
-					We explicitly clear it here since the old Document may be cached by the UA.
+			1. [=Capture the old state=] for |outboundTransition|.
 
-			1. Set |oldDocument|'s [=auto-skip view transitions=] to false.
+			1. If failure is returned, call |proceedWithNavigation| and return.
```

Comment:
Simpler to call skip the VT here? Assuming you update the skip algorithm for the case I mentioned above.